package.json

```diff
@@ -27,9 +27,11 @@
   "license": "MIT",
   "dependencies": {
     "@actions/core": "^1.10.0",
-    "@actions/exec": "^1.1.1"
+    "@actions/exec": "^1.1.1",
+    "glob": "^7.2.0"
   },
   "devDependencies": {
+    "@types/glob": "^7.2.0",
     "@types/download": "^8.0.2",
```

Comment:
I forgot to take out this one, could you do it here? I could do it but then you'd have to rebase again. :)

Reply:
@svenstaro done